### ISSUE TITLE
flux-accounting guide: missing definition(s) for terms

### ISSUE THREAD

**Issue by cmoussa1** [flux-framework/flux-docs#260]
Was perusing the flux-accounting guide this morning and noticed at least one definition missing for  **fair share**. This guide should be stepped through and updated with any other terms that would benefit from having its definition written at the bottom of the guide.

**Comment by vsoch** [comment]
@cmoussa1 this is a really good idea! I was thinking it would be useful and fun to compare the algorithms we use here with what, for example, a tool like Kueue uses. Can you keep me in the loop?

**Comment by cmoussa1** [comment]
Sure thing! When you refer to algorithm, are you referring to the algorithm that the flux-accounting priority plugin uses to calculate a job's priority?

**Comment by vsoch** [comment]
Yep, I would call "fair share" an algorithm but maybe it's not considered one always? https://en.wikipedia.org/wiki/Fair-share_scheduling

**Comment by vsoch** [comment]
And I'm worried when you say "the algorithm" that's the only one, oh no. :laughing:

**Comment by cmoussa1** [comment]
Sorry, maybe I misunderstood you and we are talking about different things! When I refer to fair share as it relates to the flux-accounting priority calculation algorithm, I mean that fair share is _one factor_ of a number of different factors that contribute to how the algorithm calculates priority.

**Comment by vsoch** [comment]
Interesting! I'd want to ask @grondo @garlick - I'm pretty sure fair share was a scheduling algorithm before it was a factor, and the design here just places it at a different level (to be part of a larger decision process). But I'm guessing it's (mostly?) the same fair share.

**Comment by garlick** [comment]
In Flux, the priority plugin (in this case provided by flux-accounting) just assigns a priority to each job.  The scheduler then takes the priority queue and creates a schedule.  So we probably wouldn't call fair share one of our scheduling algorithms.  It's just providing input to one.

**Comment by vsoch** [comment]
But is the underlying logic the same?

**Comment by garlick** [comment]
Well we're using it for a space shared system as opposed to a time shared one, and it doesn't directly determine the schedule, so that part is not the same.  The goal is the same though, to share resources fairly among users or projects as opposed to treating each individual job as an entity that has to be treated fairly.

**Comment by vsoch** [comment]
Gotcha. So I think if we are working on docs here, we can assume many readers will come with some (even minor) knowledge of what fair share is. Either we will let them make assumptions about our derivative and scope, or (what I think might be best) is we call that out, and then very clearly define the two and explain the difference.
